src/main/java/org/folio/eusage/reports/MainVerticle.java

```diff
@@ -33,6 +34,7 @@ public void start(Promise<Void> promise) {
     EusageReportsApi eusageReportsApi = new EusageReportsApi();
     RouterCreator [] routerCreators = {
         eusageReportsApi,
+        new UseOverTimeApi(),
         new Tenant2Api(eusageReportsApi),
```

Comment:
A separate YAML and implementation. Nice. A separate YAML could indicate that we make a separate interface for this end point.. But what about the remaining 2-3 entries? also separate YAML?

Reply:
Code merged into EusageReportsApi.